packages/rocketchat-importer-csv/server.js

```diff
@@ -254,7 +298,7 @@ Importer.CSV = class ImporterCSV extends Importer.Base {
 							const creator = this.getUserFromUsername(msg.username);
 							if (creator) {
 								const msgObj = {
-									_id: `csv-${ csvChannel.id }-${ msg.ts }`,
+									_id: `csv-${ csvChannel.id }-${ roomCounter }-${ msg.ts }`,
```

Comment:
I made the code doing our exports take a timestamp range as input with output always being ordered by room-specific counters. The importer code doesn't change that order per room, so the extra counter should be stable for imports done with it.
The only way I see this code duplicating data on re-import is these two cases:
1. The data being exported in a different order or file splitting, instead of the same files being used.
2. The data being a different slice of the original chats, but including a percentage of already-imported ones.
Both of these seem to me like edge cases and not very likely to happen.

Before this change, the error crashed the import entirely, reported it in the log, but didn't mark the import as failed and didn't stop the querying on the client side. This made it annoying to find (backtracking through the logs) and required the restart of the RC instance to reset the import status.
This crash is something this pull request doesn't fix. It only fixes the different causes I encountered for crashes.